apps/meteor/server/models/raw/ModerationReports.ts

```diff
@@ -24,6 +25,8 @@ export class ModerationReportsRaw extends BaseRaw<IModerationReport> implements
 			{ key: { 'message.rid': 1, 'ts': 1 } },
 			{ key: { userId: 1, ts: 1 } },
 			{ key: { 'message._id': 1, 'ts': 1 } },
+			{ key: { 'ts': 1, '_hidden': 1, 'message.u._id': 1 } },
+			{ key: { '_hidden': 1, 'reportedUser._id': 1, 'ts': 1 } },
```

Comment:
pls look into optimizing the indexes a bit, there are a lot for a rather small feature. for example, index \#3 and the last one could be compounded i believe. maybe more.